### ISSUE TITLE
MSFT-219The NVDA doesn't read some items when user try to mouse hover on them

### ISSUE THREAD

**Issue by veshu** [veshu/testjiraimport#35]
**Steps to Reproduce**
1. Navigate to https://accounts.topcoder.com/sso-login/?app=member
2. Try to mouse hover on "email or handle" text-box or "Back to Login / Privacy Policy" links or "© 2019 TOPCODER. ALL RIGHTS RESERVED" text

**Expected Result**
The NVDA should read all items when user try to mouse hover on them

**Actual Result**
The NVDA doesn't read the "email or handle" text-box or "Back to Login / Privacy Policy" links or "© 2019 TOPCODER. ALL RIGHTS RESERVED" textwhen user try to mouse hover on them

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome
**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.2.3 Audio Description or Media Alternative (Prerecorded)

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=b9c070b1%2Dbb96%2D40e1%2Db92f%2D16f47db5d911&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fdoremihong%2FTest%20for%20Accessibility%20Challenge%2FBug%5F24
